.github/workflows/task-list-check.yml

```diff
@@ -6,7 +6,6 @@ on:
       - opened
       - edited
       - reopened
-      - synchronize
```

Comment:
Why have you removed that again? We had issue to have that in, see #62

Reply:
As discussed, we check causes the build to be red all the time. We don't want to mark the checks before the changes are all done, so we removed it. @inf17101 is working on another solution here.

Reply:
After discussion with @krucod3: We remove the `synchronize` keyword.